### PR TITLE
Switches ocs-util to jupyterlab

### PR DESCRIPTION
**Pull request by jlashner**
Switches ocs-util to using jupyter lab instead of jupyter notebook.

## Description

Migrates jupyter notebook configurations to jupyter lab.

## Motivation and Context

Jupyter lab is a pretty big upgrade to jupyter notebook, and has been requested by Kenji, and is currently being used in the smurf-jupyter environment. Allows you to view other filetypes like images and csvs, run terminal sessions, and more.

## How Has This Been Tested?

Tested locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Unless I am preparing a release, I have opened this PR onto the `develop` branch.

### CODE REVIEW

File: docker/ocs-util/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 jupyter
```
Comment (BrianJKoopman) [inline docker/ocs-util/requirements.txt:1]: Do we still need `jupyter`? It doesn't look like a dependency to `jupyterlab`.
```
$ pip3 show jupyterlab
Name: jupyterlab
Version: 3.0.12
Summary: The JupyterLab server extension.
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: /home/koopman/.venv/jupyterlab/lib/python3.9/site-packages
Requires: jupyter-packaging, nbclassic, tornado, jupyter-server, jinja2, ipython, jupyter-core, jupyterlab-server, packaging
Required-by: 
```

**Review by BrianJKoopman — COMMENTED** [review]
Thanks for the switchover. I think we can drop `jupyter` from the requirements file, but correct me if I'm wrong.

**Comment by jlashner** [comment]
Just tested and you're right, it works w/o the jupyter requirement

**Review by BrianJKoopman — APPROVED** [review]
Great, thanks again!

**Pull request merged**